generators/environment/templates/outrigger/outrigger.yml

```diff
@@ -81,8 +94,22 @@ scripts:
     alias: logs
     description: Stream the logs produced by your Docker project containers. To limit output add '--tail=20'.
     run:
+      - if [ "$OSTYPE" == "linux-gnu" ]; then COMPOSE_FILE=docker-compose.yml; fi
       - docker-compose logs -ft
 
+  setup:
+    alias: setup
+    description: Run the end-to-end repository initialization and site install script.
+    run:
```

Comment:
Is the implication here is that we'll drop rig onto ci* servers? Even if we do a jenkins script wouldn't be able to call `rig project setup` due to the sync:start presence I think (unless guarded against with the linux check).

Reply:
Per the PR description, I'm still work-in-progress on refactoring the Jenkins jobs. I'm still debating whether to duplicate the essential bits-with-variations in each Jenkins job config, or to add a bin/ci-pipeline.sh script.